chain/network/src/peer.rs

```diff
@@ -731,6 +742,18 @@ impl StreamHandler<Result<Vec<u8>, ReasonForBan>> for Peer {
                 return;
             }
         };
+        // Drop duplicated messages routed within 1s
```

Comment:
one second is too much. For example, I think we re-request chunk parts every 200ms. I suggest we use a smaller number like 100ms for example

Reply:
I'm ok changing it to 100ms, we can adjust later based on real world data.